src/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Nullable>enable</Nullable>
     <BaseOutputPath>..\out</BaseOutputPath>
-    <Version>1.1</Version>
```

Comment:
Is this part of this PR?

Reply:
Investigating. Looks like i added 2 commits.

Reply:
I only added one commit, i just need to mark the "merge to" branch as `release/1.2` in GitHub. Fixed.